qa/ceph_salt/storage-profiles/bs_dedicated_db.yaml

```diff
@@ -0,0 +1,15 @@
+overrides:
+  deepsea:
```

Comment:
"deepsea" is written here intentionally?

In general, I don't know if it's a good idea to copy all of these storage profiles forward from the deepsea suite. They are very old and I doubt if they will all work in SES7.

Reply:
hmm my bad I more or less copied these from the existing deepsea ones and didn't clean them properly

Reply:
Removed storage-profiles since it's not used on ceph_salt task for now.